internal/util/test/controller_manager.go

```diff
@@ -101,7 +101,7 @@ func DeployControllerManagerForCluster(
 
 	// render all controller manager flag options
 	controllerManagerFlags := []string{
-		fmt.Sprintf("--kong-admin-url=http://%s:8001", proxyAdminURL.Hostname()),
+		fmt.Sprintf("--kong-admin-url=%s", proxyAdminURL.String()),
```

Comment:
I believe `.String()` is not necessary when you use `%s` format string.

I'll leave that to you whether you'd prefer to remove it or leave as is.

Reply:
You're right I fix it in the subsequent PR